Two more translations

MaelColuim opened the pull request:
Including comparisons

alexheretic commented:
~Isn't this pr just the other two combined?~ Oh I see this is for TW. I wonder if it was worth having both in the end, most games seem to just have simplified.

MaelColuim commented:
Yes, I did simplified comparisons, then traditional, then the traditional
other two - so the PR includes both for traditional (you can see I deleted
a PR for just the comparisons in traditional), then I went back to
Simplified and did the other sentences there.

On Thu, 1 Aug 2019, 10:13 Alex Butler, <notifications@github.com> wrote:

> Isn't this pr just the other two combined?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/big-ab-games/robo-instructus-translation/pull/49?email_source=notifications&email_token=AEMW3R7ALJAWWYN247OJ5DDQCKSNDA5CNFSM4IIK3SE2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3J5UVI#issuecomment-517200469>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEMW3RZL6LRUQNHLX4VAX3LQCKSNDANCNFSM4IIK3SEQ>
> .
>
